pymisp/api.py

```diff
@@ -307,32 +307,46 @@ def get_event(self, event: Union[MISPEvent, int, str, UUID],
         e.load(event_r)
         return e
 
-    def add_event(self, event: MISPEvent, pythonify: bool = False) -> Union[Dict, MISPEvent]:
+    def event_exists(self, event: Union[MISPEvent, int, str, UUID]) -> bool:
+        """Fast check if event exists.
+
+        :param event: Event to check
+        """
+        event_id = get_uuid_or_id_from_abstract_misp(event)
+        r = self._prepare_request('HEAD', f'events/view/{event_id}')
+        if r.status_code in (404, 200):
+            return r.status_code == 200
+        self._check_response(r)
```

Comment:
I'm not sure about the call to `self._check_response` here. I think the method should return `False` if the `status_code` is 404, and maybe return an exception otherwise? 

`self._check_response` does a lot of massaging on the data that doesn't make any sense if the response body is empty anyway.

Reply:
Correct, fixed now.

Reply:
Thanks!